onnx/defs/tensor/defs.cc

```diff
@@ -296,8 +296,11 @@ ONNX_OPERATOR_SET_SCHEMA(
             if (!ctx.getInputType(0)->tensor_type().has_shape()) {
               return;
             }
-            const auto& splitDim =
-                ctx.getInputType(0)->tensor_type().shape().dim(axis);
+            const auto& shape = ctx.getInputType(0)->tensor_type().shape();
+            if (axis >= shape.dim_size()) {
+                return;
```

Comment:
you should throw error here instead of silently return

Reply:
Fixed.